assets/js/steps.js

```diff
@@ -0,0 +1,35 @@
+import accessibleAutocomplete from './accessible-autocomplete.min.js'
```

Comment:
If you want to remove extension correct ones is import accessibleAutocomplete from './accessible-autocomplete.min'
2024-07-05 09:53:21 [ESBuild] ✘ [ERROR] Could not resolve "./accessible-autocomplete"
2024-07-05 09:53:21 [ESBuild] 
2024-07-05 09:53:21 [ESBuild]     assets/js/steps.js:1:35:
2024-07-05 09:53:21 [ESBuild]       1 │ import accessibleAutocomplete from './accessible-autocomplete'

Reply:
I don't think this is right? Shouldn't this be a reference to the package rather than the static file in the directory? i.e. the line should be as Thomas puts it - with no `./` at the start and no `.min.js` at the end - in fact do we need the .js and .css files in the repo at all? Shouldn't they get bundled in via esbuild?

Reply:
unfortunately this package was developed originally as react package and later forked manually to GDS backend.
Ideally we should create nunjuck macro which will do all the mentioned things under the hood.